src/kv/kv_types.h

```diff
@@ -797,8 +797,8 @@ struct formatter<kv::Configuration::Nodes>
   }
 
   template <typename FormatContext>
-  auto format(const kv::Configuration::Nodes& nodes, FormatContext& ctx)
-    -> decltype(ctx.out())
+  auto format(const kv::Configuration::Nodes& nodes, FormatContext& ctx) const
+    -> decltype(ctx.out()) const
```

Comment:
```suggestion
  auto format(const kv::Configuration::Nodes& nodes, FormatContext& ctx) const
    -> decltype(ctx.out())
```

Second const alters the return type, not the method.